packages/infra/lib/api-stack.ts

```diff
@@ -448,6 +448,9 @@ export class APIStack extends Stack {
         apiURL: apiService.loadBalancer.loadBalancerDnsName,
         envType: props.config.environmentType,
         sentryDsn: props.config.lambdasSentryDSN,
+        dbCredsSecret,
+        dbCluster,
+        dbReadOnlyEndpoint: dbCluster.clusterReadEndpoint,
```

Comment:
passing read only endpoint

Reply:
If we're passing the cluster, might as well not send the read only endpoint and let the downstream function get it from the cluster.

OTOH, instead of passing the cluster - which is very permissive, we could just pass the readonly endpoint like you suggested and `dbCluster.connections`, that would be received as `dbConnections: IConnectable`, then we could:

```ts
dbConnections.connections.allowFrom(...)
```